src/lava/utils/profiler.py

```diff
@@ -12,3 +12,125 @@
 elementary operation has a defined execution time and energy cost, which is
 used in a performance model to calculate execution time and energy.
 """
```

Comment:
Docstring: 
will contain -> contains [otherwise, this needs to be rewritten once it works next month, and we will forget it]
power and performance of workloads -> isn't power an aspect of performance?

Maybe a few lines on what the Profiler simulation is good for? 
- Simulate a network on CPU before it is being deployed - in which situations is that feature helpful? If people don't have access to Loihi yet. If they want to understand the performance on a fine-granular layer. ...

Reply:
I moved this Docstring to the Profiler class docstring without the "will".